<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CostsCalculator</class>
 <widget class="QDialog" name="CostsCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpin_FueConsuption">
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_CostKilometer">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButton_Calculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="pushButton_Exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEdit_Insurance"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_Maintance">
     <property name="text">
      <string>Maintance / year</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_FuelPrice">
     <property name="text">
      <string>Fuel price / liter</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_kilometers">
     <property name="text">
      <string>Driven kilometer / year</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_CarModel">
     <property name="text">
      <string>model</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_driven">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="lineEdit_Maintance"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_CarHorsepower">
     <property name="text">
      <string>horsepower</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_CarYear">
     <property name="text">
      <string>year</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_FuelPer100">
     <property name="text">
      <string>Consuption / 100 km</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_CarMake">
     <property name="text">
      <string>make</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_inspection">
     <property name="text">
      <string>Inspection / year</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_Insurance">
     <property name="text">
      <string>Insurance / year</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpin_FuelPriceLiter">
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>1.600000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="lineEdit_Inspection"/>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="label_CostTotal">
     <property name="text">
      <string>total</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_CostConsuption">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
